### ISSUE TITLE
(UAT) Unable to delete Partner

### ISSUE THREAD

**Issue by KKaselkar** [unicef/etools-issues#757]
Not able to delete a partner that has draft PCA (Imported from Jordan with a partner in Pakistan)

**Comment by ewheeler** [comment]
@KKaselkar Removing priority for now.  priorities are added during our weekly prioritization meetings. if you find something you believe needs urgent attention, please escalate to Shrini and myself by email. Removing priority for now.

**Comment by ewheeler** [comment]
@KKaselkar is this related to any of the existing issues related to partner deletion? If so, please update the appropriate card rather than opening a new one

#368
#503
#367
#594
#604
#705

**Comment by KKaselkar** [comment]
None of the above issues cover this new issue 757.
In this new issue a partner was added by mistake from another work space and then user was not able to delete him as there no option.

**Comment by ewheeler** [comment]
sounds like this could be related to #705 but you'll have to provide enough detail for someone to reproduce before we can be certain

**Comment by KKaselkar** [comment]
Steps to reproduce in Pakistan workspace:
1) Add a new Partner (e.g. 2500234857) in Pakistan
2) Create a draft PCA
3) Goto the partner record and hit the delete button
System doesnt allow to delete the record though PCA is in draft status.
It should not allow to delete if the partner has cash transfer or assigned agreement but it should allow to delete the partner record if its in draft mode.
